.prettierrc.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   arrowParens: 'avoid',
   overrides: [
     {
-      files: ['*.code-workspace'],
+      files: ['*.code-workspace', '*.json'],
```

Comment:
drive-by to allow force-formatting of `.json` files. They used to be handled by Hermes parser which can't parse .json files.